scanpipe/pipes/__init__.py

```diff
@@ -180,6 +180,7 @@ def update_or_create_package(project, package_data, codebase_resources=None):
     package_data = _clean_package_data(package_data)
     # No values for package_uid requires to be empty string for proper queryset lookup
     package_uid = package_data.get("package_uid") or ""
+    datasource_id = package_data.get("datasource_id")
```

Comment:
Looking at the comment above:
"# No values for package_uid requires to be empty string for proper queryset lookup"

Could you make sure this is not also required for the `datasource_id` field?